cmd/gobl.xinvoice/convert.go

```diff
@@ -47,27 +49,47 @@ func (c *convertOpts) runE(cmd *cobra.Command, args []string) error {
 	}
 	defer out.Close() // nolint:errcheck
 
-	inData, err := io.ReadAll(input)
+	in, err := io.ReadAll(input)
 	if err != nil {
 		return fmt.Errorf("reading input: %w", err)
 	}
-	env := new(gobl.Envelope)
-	if err := json.Unmarshal(inData, env); err != nil {
-		return fmt.Errorf("parsing input as GOBL Envelope: %w", err)
-	}
+	j := json.Valid(in)
+	var o []byte
+	if j {
+		env := new(gobl.Envelope)
+		if err := json.Unmarshal(in, env); err != nil {
+			return fmt.Errorf("parsing input as GOBL Envelope: %w", err)
+		}
+		switch c.format {
+		case "xrechnung-cii":
```

Comment:
Just a note for the future, we might need to add version information here so its clear what output will be produced.